en_US/advanced/quic.md

```diff
@@ -0,0 +1,80 @@
+# QUIC transport support
+
+QUIC is new general-purpose transport layer protocol for modern network.
+
+## Introduction
+
+MQTT over QUIC protocol is an experiment feature in EMQX 5.0.
+
+You can play with MQTT over QUIC to verify how it could improve network connectivity between Client and EMQX comparing to TLS1.2 and TLS. 
+
+note, this is a feature of MQTT over QUIC not MQTT over HTTP/3.
+
+## Features
+
+- Support 0-RTT handshake
+
+  For low latency connection establishments.
+  
+- Support Client Address Migration
+
+  MQTT (over QUIC) client could change its network layer address or transport layer port without 
+  any traffic disturbance. For example, in the scenario of NAT rebinding.
+  
+- Disabled traffic encryption
+
+  MQTT (over QUIC) client could run unencrypted traffic after QUIC TLS1.3 handshake is done.
+  
+  see: https://datatracker.ietf.org/doc/html/draft-banks-quic-disable-encryption
+  
+  note: this is a draft rfc spec.
+
+## Implementation
+
+### Client
+
+MQTT Client (QUIC client) initials a QUIC connection to EMQX (QUIC Server).
+
+After the conneciton handshake is done, MQTT Client then starts a bidirectional stream to EMQX.
+
+Then all the MQTT Messages will be exchanged on this bidirectional stream.
+
+Either side of the stream closing the stream will lead to conneciton close just like TCP connection close.
+
+Reference client:
+
+https://github.com/emqx/emqtt
+
+### For TLS 1.3 handshake
+
+ALPN: MQTT
+
+## Feature Enabling
+
+QUIC transport listener is diabled by default.
+
+To experiment with QUIC transport, you need to 
+
+1. Enable it via configuration 
+
+```
+listeners.quic.default {
+  enabled = true
+  bind = "0.0.0.0:14567"
+  max_connections = 1024000
+  keyfile = "{{ platform_etc_dir }}/certs/key.pem"
+  certfile = "{{ platform_etc_dir }}/certs/cert.pem"
+}
+```
+
+note, `{{ platform_etc_dir }}` is the emqx etc dir something like `/etc/emqx/certs`
+
+1. restart EMQX.
```

Comment:
I think it might become 1. in the rendered document, because the list items in markdown have to be indented.